fix: normalize init script path for all args

juanamari94 opened the pull request:
- [X] Ready for review
- [X] Follows CONTRIBUTING rules
- [X] Reviewed by Snyk internal team

#### What does this PR do?

Normalized init script path is sent for bot dependency discovery and call graph building.
Disables explicit propagation of the `init script` argument for dependency discovery.

This pull request was made in tandem with @pwnslinger . See: https://github.com/snyk/snyk-gradle-plugin/pull/170

#### Where should the reviewer start?


#### How should this be manually tested?


#### Any background context you want to provide?

Using different paths for init script and build.gradle files would create an error on the java call graph builder side if the parent directories differed.


#### What are the relevant tickets?


#### Screenshots


#### Additional questions

anthogez commented on test/system/plugin.test.ts:
```diff
@@ -112,7 +112,7 @@ test('run inspect() with reachableVulns', async (t) => {
       path.join('.', rootNoWrapper),
       'gradle',
-      path.join(rootNoWrapper, 'init.gradle'),
+      "'" + path.join(rootNoWrapper, 'init.gradle') + "'", // arg should be normalized with quotes
```
is it handling both windows/unix?

Commit:
fix: dep discovery init script propagation, path normalization

Normalizes the initialization script (init-script) path for gradle for both dependency discovery as well as building the call graph.
Eliminates explicit initialization script for dependcy discovery.

snyksec commented:
:tada: This PR is included in version 3.13.2 :tada:

The release is available on:
- [npm package (@latest dist-tag)](https://www.npmjs.com/package/snyk-gradle-plugin/v/3.13.2)
- [GitHub release](https://github.com/snyk/snyk-gradle-plugin/releases/tag/v3.13.2)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
